Xamarin.Forms.Platform.iOS/CollectionView/CollectionViewController.cs

```diff
@@ -42,7 +42,14 @@ public override UICollectionViewCell GetCell(UICollectionView collectionView, NS
 
 		public override nint GetItemsCount(UICollectionView collectionView, nint section)
 		{
-			return _itemsSource.Count;
+			var count = _itemsSource.Count;
+			if (_layout.IsDeterminingCellSize && count == 1)
+			{
+				_layout.FinishedDeterminingCellSize();
```

Comment:
There appears to be a disconnect here and the layout. It is `FinishedDeterminingCellSize` here, but `FinishDetermineCellSize` in the layout. Maybe this?

```suggestion
				_layout.FinishDeterminingCellSize();
```